code/game/machinery/vending/vendor_types/crew/vehicle_crew.dm

```diff
@@ -386,25 +390,29 @@ GLOBAL_LIST_INIT(cm_vending_clothing_vehicle_crew, list(
 	)
 
 /obj/effect/essentials_set/tank/tankflamer
+	desc = "A small LZR-N Flamer Unit. Essentially a modified version of your bog standart flamer."
 	spawned_gear_list = list(
 		/obj/item/hardpoint/secondary/small_flamer,
 		/obj/item/ammo_magazine/hardpoint/secondary_flamer,
 	)
 
 /obj/effect/essentials_set/tank/tow
+	desc = "A quint rocket launcher capable of firing four rockets in quick succession."
```

Comment:
```suggestion
	desc = "A quad rocket launcher capable of firing four rockets in quick succession."
```

Reply:
It is a quint launcher, it 5 rocket loaded.

Reply:
huh. fair